apps/test/unit/lib/ui/CheckedRadioButtonTest.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
```

Comment:
sorry to be a pain, but we're trying to move away from `chai-enzyme` which lets you do some of the `expect(...).to.be....` syntax you're using below. please change this line from `configuredChai` to `reconfiguredChai`, here and in other files, and then fix any syntax errors you get as a result. 

FYI @jmkulwik

Reply:
+1
Quick comment though. It's not completely necessary to remove chai-enzyme in order to upgrade to React 16, but we are working to deprecate it. If it's already included you don't have to remove it (although bonus points if you do), but don't add it if it isn't already in a test file. Use reconfiguredChai instead.

Reply:
example: https://github.com/code-dot-org/code-dot-org/pull/27553